nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -25,6 +31,7 @@ async def new_connection() -> AsyncIterator[Connection]:
             "aes128-ctr",
         ],
         compression_algs=None,
+        errors="replace",
```

Comment:
This hides error for sure and this will lead to instabilities. I remember me and @lcruz99 or @arg0d having problems with this option set. But can't really recall what it was

Reply:
The problem now is there is some ssh error related to encoding when writing logs to file with the new uniffi+pyro5 system. I can change to `ignore` if that makes more sense to you, but using `strict` will fail all/most windows tests and a handful of macos tests. Honestly, I don't understand why it's happening since the log lines aren't actually being passed over ssh. I had an idea to run the remote libtelio instance in a detached shell, which might fix it. I'll try that see if it works better

Reply:
Ah yes, this is exact problem it was hiding. We didn't want to trip over of some missing logs, due to encoding errors being suppressed

Reply:
Maybe this [fix](https://github.com/NordSecurity/rust_build_utils/pull/2/commits/206453391abd30e1891a19917f3843a48bab4cdd) would help if we would prepend all ssh_process commands and we could leave errors as default.

Reply:
That fix didn't work, but simply setting the encoding to UTF-8 when opening the file did work